packages/dashboard/src/components/composed/Txs/Tx/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, createStyles } from "@mantine/core";
+import { useDisclosure } from "@mantine/hooks";
+import { ChevronRight } from "react-feather";
+import type { ReceivedMessageLifecycle } from "@truffle/dashboard-message-bus-client";
+import type { DashboardProviderMessage } from "@truffle/dashboard-message-bus-common";
+import TxModal from "src/components/composed/Txs/Tx/TxModal";
+
+const useStyles = createStyles((theme, _params, getRef) => ({
+  root: {
+    [`&:hover .${getRef("rightIcon")}`]: {
+      opacity: 1,
+      color: theme.colors["truffle-teal"][7]
+    }
+  },
+  label: {
+    fontFamily: theme.fontFamilyMonospace
+  },
+  rightIcon: {
+    ref: getRef("rightIcon"),
+    opacity: 0.2,
+    transition: "opacity 0.2s"
+  }
+}));
+
+type TxProps = {
+  lifecycle: ReceivedMessageLifecycle<DashboardProviderMessage>;
+};
+
+function Tx({ lifecycle }: TxProps): JSX.Element {
```

Comment:
Maybe better not to abbreviate this? Existing style is that we don't abbreviate anything, ever.

Reply:
I've [removed all abbreviations (except for "RPC")](https://github.com/trufflesuite/truffle/commit/9ad1ad3500d54161ed10028d9bf252b0cee50d23), some file / dir names were changed.